### DISCUSSION TITLE
How may I go about making http3 requests with resty?

### DISCUSSION THREAD

**Discussion by rew1nter** [go-resty/resty#791] [Q&A] [answered]
quic-go transport is probably not usable with resty. Is there any other recommended way?

**Comment by jeevatkm** [comment, marked as answer]
@rew1nter Go lang has not yet added HTTP3 to the standard library; I will have to wait for that. 
However, based on the [quic-go](https://quic-go.net/docs/http3/client/) library, there is a possibility to use quick-go lib with Resty.

Reference: https://quic-go.net/docs/http3/client/
```go
roundTripper := &http3.RoundTripper{
	TLSClientConfig: &tls.Config{},  // set a TLS client config, if desired
	QUICConfig:      &quic.Config{}, // QUIC connection options
}

client := resty.New().SetTransport(roundTripper)

// then typical use of resty happens
```
Give it a try.
